Multisig/Logic/Models/UInt256String.swift

```diff
@@ -27,7 +27,7 @@ struct UInt256String: Hashable, Codable {
         let container = try decoder.singleValueContainer()
         if let string = try? container.decode(String.self) {
             if string.hasPrefix("0x") {
-                let data = Data(hex: string)
+                let data = Data(ethHex: string)
```

Comment:
can we check why this changed?

Reply:
I fixed it with a separate commit when checking how wallet connect requests work. I assume the issue was that when dapp was sending safeTxGas / nonce in hex data format, the displayed number in UI was wrong